ansible/roles/de-update-integration-data/tasks/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: update docker-py
+  pip: name="docker-py" state="latest"
+
+- name: pull the latest integrationator image
+  docker_image: name="{{ integrationator_repo }}" tag="{{ docker_tag }}" force=True
+
+- name: update integration data records
+  shell: >-
+    docker run --rm {{ integrationator_repo }}:{{ docker_tag }} -l {{ ldap_host }} -h {{ db_host }}
+    -p {{ db_port }} -d {{ db_name }} -U {{ db_user }} -P {{ db_password }}
```

Comment:
I tried using `docker wait` without `wait_for`, but I couldn't get that to work. I didn't think that `wait_for` would be necessary since `docker wait` is supposed to block until the container stops. Besides, since this adds a shell command, I think that using the shell command for `docker run` is reasonable.